README.md

```diff
@@ -62,6 +62,12 @@ Starting is easy as 1-2-3
   <img src="https://contrib.rocks/image?repo=Blueforcer/awtrix-light" />
 </a>
 
+The project is built with [PlatformIO](https://platformio.org):
```

Comment:
No need to explain platformIO basics in the welcome readme. 
AWTRIX is mainly for non tech savy users. 99% of the users doesnt need and want to build the code.

Reply:
It was not obvious to me, and more info does not hurt, but OK I've removed it.